spec/support/solargraph_helpers.rb

```diff
@@ -74,4 +76,20 @@ def completion_pins_at(filename, position, map = api_map)
   def completion_at(filename, position, map = api_map)
     completion_pins_at(filename, position, map).map(&:name)
   end
+
+  # Expect that a local can be inferred with +expected_type+
+  #
+  # @param var_name [String]
+  # @param expected_type [String]
+  # @param file [String] The filename (defaults to filename defined in test)
+  # @param map [Solargraph::ApiMap] The Api Map (defaults to the one defined in a test)
+  #
+  # @return [Solargraph::Pin::BaseVariable] The variable pin
+  def expect_local_type(var_name, expected_type, file = filename, map = api_map)
```

Comment:
Nice helper 👍 

suggestion: can we please rename it to
```suggestion
  def expect_local_variable_type(var_name, expected_type, file = filename, map = api_map)
```

"local_type" sounds a bit misleading.